fabric-network/bc-network.sh

```
@@ -0,0 +1,24 @@
+# Create channel and join peers
+docker exec cli peer channel create -o orderer.example.com:7050 -c healthcare -f ./channel-artifacts/channel.tx
+docker exec cli peer channel join -b healthcare.block
+
+# Deploy chaincode
+docker exec cli peer lifecycle chaincode package healthcare.tar.gz --path chaincode --lang golang --label healthcare_1
+
+docker exec cli peer lifecycle chaincode install healthcare.tar.gz
+
+docker exec cli peer lifecycle chaincode approveformyorg \
+  --channelID healthcare --name healthcare --version 1.0 \
+  --package-id $(docker exec cli peer lifecycle chaincode queryinstalled | awk '/healthcare_1/ {print $3}' | sed 's/,//') \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)